db/typings.d.ts

```diff
@@ -0,0 +1,6 @@
+import type { Pool } from 'pg'
+
+export interface pgPools {
+  pgPool: Pool;
+  pgPoolEvaluate: Pool;
+}
```

Comment:
Great to have the types! 💪🏻 

I think you are using different property names in the JS code above.

```suggestion
export interface pgPools {
  stats: Pool;
  evaluate: Pool;
}
```